test/functionalTest/cases/1112_metadata_as_keymap/metadata_with_dots_v1.test

```diff
@@ -171,7 +171,7 @@ Content-Length: 269
 
 03. Check the '=' in DB
 =======================
-{ "attrs" : { "A" : { "type" : "T", "creDate" : REGEX(.*), "value" : "foo", "md" : { "m=x" : { "type" : "string", "value" : "black" }, "m=y" : { "type" : "string", "value" : "sweet" } }, "mdNames" : [ "m.x", "m.y" ] } } }
+{ "attrs" : { "A" : { "value" : "foo", "type" : "T", "creDate" : REGEX(.*), "md" : { "m=x" : { "type" : "string", "value" : "black" }, "m=y" : { "type" : "string", "value" : "sweet" } }, "mdNames" : [ "m.x", "m.y" ] } } }
```

Comment:
This change (and the other in the _v2.test file) are due to the changes in the moment in which valueBson() is called in src/lib/mongoBackend/MongoCommonUpdate.cpp. Now `value` is calculated before than `type`, so this change. However, it doesn't have any actual impact (as we used the usual semantic in JSON under which the key order is not relevant).

NTC (just informative)